fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1217,6 +1217,15 @@ public class Config extends ConfigBase {
     @ConfField
     public static boolean enable_auth_check = true;
 
+    /**
+     * If set to true, auth check for StarRocks external table will be enabled. The check
+     * only happens on the target cluster. The default is false to ensure users can upgrade
+     * StarRocks smoothly. Otherwise, the upgrade can fail if the existed external table has
+     * wrong user/password, and the authentication is checked in the new version of StarRocks.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_starrocks_external_table_auth_check = false;
```

Comment:
This a feature directly related to users, please remember to update the main branch of the documentation repo(or previous branch if you plan to cherry-pick this pr to older branch).

Reply:
Thanks for reminding.